Samples/Client/DirectxWin32/src/default_main_window.cpp

```diff
@@ -689,11 +689,11 @@ void DefaultMainWindow::LayoutConnectUI(bool show)
 	} windows[] =
 	{
 		{ auth_uri_label_, L"Auth Uri"},
-		{ auth_uri_, auth_uri_val_.c_str()},
+		{ auth_uri_,L"XXXyyyYYYgggXXXyyyYYYgggXXXyyyYYYggggggXXXyyyYYYgggXXXyyyYYYgggXXXyy" },
 		{ auth_code_label_, L"Auth Code" },
-		{ auth_code_, auth_code_val_.c_str()},
+		{ auth_code_, L"XXXyyyYYYgggXXXyy" },
 		{ label1_, L"Server" },
-		{ edit1_, L"XXXyyyYYYgggXXXyyyYYYgggXXXyyyYYYggg" },
+		{ edit1_, L"XXXyyyYYYgggXXXyyyYYYgggXXXyyyYYYggggggXXXyyyYYYgggXXXyyyYYYgggXXXyy" },
```

Comment:
does making this change back affect the layout? i imagined that `CalculateWindowSizeForText` below would be inaccurate if we don't have the actual text here.

If this works (as i assume it does) then we can remove the `auth_code_val_` variable all together, and just call the `::SetWindowText()` functions as you already do in the setters